src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -903,6 +903,23 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             return;
         }
 
+        if (vkey == VK_SPACE && modifiers.IsAltPressed())
+        {
+            if (const auto bindings = _settings.KeyBindings())
+            {
+                if (!bindings.IsKeyChordExplicitlyUnbound({ modifiers.IsCtrlPressed(), modifiers.IsAltPressed(), modifiers.IsShiftPressed(), modifiers.IsWinPressed(), vkey, 0 }))

```

Comment:
Similarly to `_TryHandleKeyBinding` you should pass the scanCode here if you can.

Reply:
Ah yes, thanks for the catch!